src/gradata/hooks/brain_maintain.py

```
@@ -0,0 +1,68 @@
+            if md_file.name == "lessons.md":
+                continue
+            try:
+                text = md_file.read_text(encoding="utf-8")
+                fts_index(md_file.name, "markdown", text, ctx=ctx)
+            except Exception:
+                continue
+    except Exception:
+        pass
+
+
+def _generate_manifest(brain_dir: str, ctx=None) -> None:
```

Ruff (0.15.9)

[warning] Unused function argument: `brain_dir`

(ARG001)